### PR TITLE
StudentGuardian Service Delete By Id 

### PR DESCRIPTION
**Pull request by JaAfghan**

### CODE REVIEW

**Comment by AgentEnder** [comment]
@JaAfghan you've got some conflicts that'll need to be fixed

**Comment by hassanhabib** [comment]
@JaAfghan please address conflicts.
@viralpandya please take a look at this PR.

**Comment by hassanhabib** [comment]
@JaAfghan one more conflict.

**Comment by tajwal** [comment]
> @JaAfghan one more conflict.

@JaAfghan please fix the conflict am waiting for last controller action.

**Comment by JaAfghan** [comment]
> @JaAfghan one more conflict.

Conflict Resolved

**Comment by JaAfghan** [comment]
> > @JaAfghan one more conflict.
> 
> @JaAfghan please fix the conflict I am waiting for last service action to complete my controller.

Conflict Resolved

**Comment by hassanhabib** [comment]
> > > @JaAfghan one more conflict.
> > 
> > 
> > @JaAfghan please fix the conflict I am waiting for last service action to complete my controller.
> 
> Conflict Resolved

@JaAfghan you have some build issues.

**Comment by JaAfghan** [comment]
> > > > @JaAfghan one more conflict.
> > > 
> > > 
> > > @JaAfghan please fix the conflict I am waiting for last service action to complete my controller.
> > 
> > 
> > Conflict Resolved
> 
> @JaAfghan you have some build issues.

@hassanhabib build issues is solved.

File: OtripleS.Web.Api.Tests.Unit/Services/StudentGuardianServiceTests/StudentGuardianServiceTests.Exceptions.Delete.cs [hunk 1]
```diff
@@ -0,0 +1,173 @@
...
+        {
+            // given
+            Guid randomStudentGuardianId = Guid.NewGuid();
+            Guid inputStudentGuardianId = randomStudentGuardianId;
+            Guid randomStudentId = Guid.NewGuid();
+            Guid inputStudentId = randomStudentId;
+            SqlException sqlException = GetSqlException();
+
+            var expectedStudentGuardianDependencyException
+                = new StudentGuardianDependencyException(sqlException);
```
Comment (hassanhabib) [inline OtripleS.Web.Api.Tests.Unit/Services/StudentGuardianServiceTests/StudentGuardianServiceTests.Exceptions.Delete.cs:32]: Move the equal sign above

Comment (JaAfghan) [reply OtripleS.Web.Api.Tests.Unit/Services/StudentGuardianServiceTests/StudentGuardianServiceTests.Exceptions.Delete.cs]: Done

Comment (tajwal) [reply OtripleS.Web.Api.Tests.Unit/Services/StudentGuardianServiceTests/StudentGuardianServiceTests.Exceptions.Delete.cs]: Mr. @hassanhabib can you please review & merge this action of service, as I am waiting for this.

**Pull request merged**